### PR TITLE
#188 Expand /status endpoint with more information 

### PR DESCRIPTION
**Pull request by linaskitm**

### CODE REVIEW

File: src/getVersion.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import fs from 'fs';
+
+export default async function getVersion() {
+    const result = await fs.promises.readFile('package.json', 'utf8');
```
Comment (psaulenas) [inline src/getVersion.ts:4]: You could get the version using import from `package.json`
```js
import { version } from '../package.json';
```

Then probably this separate file (function) not needed

Comment (linaskitm) [reply src/getVersion.ts]: Thanks for review, I refactored the code, but I had to add  in tsconfig.json "resolveJsonModule": true

Comment (psaulenas) [reply src/getVersion.ts]: That is fine 👍

**Comment by ignasposka** [comment]
Sorry, but this feature was merged from another PR.

**Pull request closed without merging**